features/projects/api/use-get-projects.tsx

```diff
@@ -1,7 +1,25 @@
 import { useQuery } from "@tanstack/react-query";
 import { getProjects } from "@api/projects";
-import type { Project } from "@api/projects.types";
+import { ProjectStatus, type Project } from "@api/projects.types";
+
+// Converts project status data from 'info' to 'stable' and 'error' to 'critical'
+function transformStatus(status: string): ProjectStatus {
+  switch (status) {
+    case "info":
+      return ProjectStatus.stable;
+    case "error":
+      return ProjectStatus.critical;
+    default:
+      return status as ProjectStatus;
+  }
+}
 
 export function useGetProjects() {
-  return useQuery<Project[], Error>(["projects"], getProjects);
+  return useQuery<Project[], Error>(["projects"], getProjects, {
+    select: (data) =>
+      data.map((project) => ({
+        ...project,
+        status: transformStatus(project.status),
```

Comment:
While this works I wouldn't recommend changing the status here. I like to separate the domain or core data models (in this case a Project) from the UI presentation (in this case e.g. that "Critical" is shown to the user when the `project.status` is `error`.

So basically from my perspective the word "Critical" is only the visual representation of project status "error". This visual representation only concerns the component (in this case `ProjectCard`).

So instead of doing the transformation here I'd introduce a mapper object like 

```
const statusTexts = {
  [ProjectStatus.info]: "Stable",
  [ProjectStatus.warning]: "Warning",
  [ProjectStatus.error]: "Critical",
};
```

in the component and then render the text like `statusTexts[project.status]`.

I hope that makes sense. Let me know if you have any questions